### PR TITLE
Launch RPC gateway for Optimism with new providers

### PR DESCRIPTION
**Pull request by rollingtumbling**
100% launch on Optimism traffic.
- Infura: 100% traffic
- Quicknode: 0% traffic, shadowing to collect latency metrics
- NirvanaLabs: 0% traffic, shadowing to collect latency metrics
- Alchemy: 0% traffic, shadowing to collect latency metrics

Also add related Optimism dashboard.

Manually tested on local environment.

### CODE REVIEW

**Comment by rollingtumbling** [comment]
* **#497** <a href="https://app.graphite.dev/github/pr/Uniswap/routing-api/497?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rollingtumbling and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/config/rpcProviderProdConfig.json [hunk 1]
```diff
@@ -3,6 +3,12 @@
     "chainId": 43114,
     "useMultiProviderProb": 1,
     "providerInitialWeights": [9, 1, 0],
-    "providerUrls": ["WEB3_RPC_43114", "WEB3_RPC_43114_QUICKNODE", "WEB3_RPC_43114_NIRVANA"]
+    "providerUrls": ["WEB3_RPC_43114_INFURA", "WEB3_RPC_43114_QUICKNODE", "WEB3_RPC_43114_NIRVANA"]
+  },
+  {
+    "chainId": 10,
+    "useMultiProviderProb": 1,
+    "providerInitialWeights": [1, 1, 1, 1],
```
Comment (cgkol) [inline lib/config/rpcProviderProdConfig.json:11]: you want to equally distribute between them? Or do we want to do 100% infura and just run the rest in shadow? i thought the latter

Comment (rollingtumbling) [reply lib/config/rpcProviderProdConfig.json]: Nice catch. Fixed as suggested.

**Review by cgkol — COMMENTED** [review]
How do we know that the shadow will be triggered? is it by default because the url providers are passed in?

**Comment by rollingtumbling** [comment]
> How do we know that the shadow will be triggered? is it by default because the url providers are passed in?

It's enabled by default if we have it in `rpcProviderProdConfig.json`.

**Pull request merged**